common-utils/src/main/java/com/stnetix/ariaddna/commonutils/xmlparser/handlers/StorageDBHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.stnetix.ariaddna.commonutils.xmlparser.handlers;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+/**
+ * Created by alexkotov on 30.05.17.
+ */
+public class StorageDBHandler extends DefaultHandler {
```

Comment:
The name of this class is misleading, please rename it. 
Also in class name please use camel case convention.
For example: XmlDbSettingHandler